Formula/coredns.rb

```diff
@@ -0,0 +1,83 @@
+class Coredns < Formula
+  desc "DNS server that chains plugins"
+  homepage "https://coredns.io/"
+  url "https://github.com/coredns/coredns/archive/v1.6.9.tar.gz"
+  sha256 "e100a946d5d60d936cb313bdb04fd96162d5a7229e08887ad497660acee4b36f"
+  head "https://github.com/coredns/coredns.git"
+
+  depends_on "go" => :build
+
+  def default_config
+    <<~EOS
+      . {
+        hosts {
+          fallthrough
+        }
+        forward . 1.1.1.1 1.0.0.1 8.8.8.8 8.8.4.4 9.9.9.9 149.112.112.112
+        cache
+        errors
+        log
+      }
+    EOS
+  end
+
+  def install
+    system "make"
+    (buildpath/"Corefile.example").write default_config
+    (etc/"coredns").mkpath
+    etc.install "Corefile.example" => "coredns/Corefile"
+    bin.install "coredns"
+  end
+
+  def caveats
+    <<~EOS
+      Config file: #{etc}/coredns/Corefile
+      DNS log file: #{var}/log/coredns.log
+      Start service: sudo brew services start coredns
+      Activate DNS: networksetup -setdnsservers Wi-Fi 127.0.0.1
+      Test functionality: dig google.com @127.0.0.1
+
+      DNS proxy servers:
+
+      Cloudflare -> https://1.1.1.1/
+      Google DNS -> https://dns.google.com/
+      Google DNS -> https://developers.google.com/speed/public-dns
+      IBM Threat -> https://www.quad9.net/faq/
```

Comment:
This should all be in the upstream documentation.

Reply:
I have removed the unofficial information with [commit 62f7878](https://github.com/Homebrew/homebrew-core/pull/53755/commits/62f7878882bc06384af77103896dc5b00eda4321). Users who want to use the DNS server as the entry point for every DNS query in their system will need to search online how to configure the macOS network settings to route all network traffic through CoreDNS.